acf-json/group_6469c5e35b44c.json

```diff
@@ -89,21 +89,92 @@
                     "append": "",
                     "parent_repeater": "field_6469c5e3696cf"
                 },
+                {
+                    "key": "field_661004250bd94",
+                    "label": "Content Type",
+                    "name": "content_type",
+                    "aria-label": "",
+                    "type": "radio",
+                    "instructions": "",
+                    "required": 0,
+                    "conditional_logic": 0,
+                    "wrapper": {
+                        "width": "",
+                        "class": "",
+                        "id": ""
+                    },
+                    "gutenberg_visibility": "both",
+                    "relevanssi_exclude": 0,
+                    "wpml_cf_preferences": 0,
+                    "choices": {
+                        "card-strapline": "Strapline (Bold)",
+                        "card-excerpt": "Excerpt"
+                    },
+                    "default_value": "card-strapline",
+                    "return_format": "value",
+                    "allow_null": 0,
+                    "other_choice": 0,
+                    "layout": "vertical",
+                    "save_other_choice": 0,
+                    "parent_repeater": "field_6469c5e3696cf"
+                },
                 {
                     "key": "field_6469c5e3f045f",
                     "label": "Strapline",
-                    "name": "strapline",
+                    "name": "card_strapline",
```

Comment:
@rachaelob Sorry I hadn't noticed this before, the strapline field name must not be changed, doing so means any existing strapline data cannot be used. In addition, it doesn't need to be namespaced (the `card_` bit), as it's clear what this belongs to.

Can you remove `card_` from both fields, and update any references to this also.